app/controllers/events_controller.rb

```diff
@@ -3,6 +3,11 @@ class EventsController < ApplicationController
 
   def show
     @event = find_event(params[:id])
+    @donations = if params[:search]
```

Comment:
Now this action has two instance variables. For our projects in Tinkerbox, when that happens, we introduce another object that contains them, but since we haven't gone through that yet, this is good. 😀